graylog-configurer

```diff
@@ -18,18 +20,18 @@ set_input()
 
   input_id=$(curl -s -XGET "${graylog_api}/system/inputs" | jq -r '.inputs[] | select(.title == "'"${input_name}"'") | .id')
   if [ ! "${input_id}" ]; then
-    curl -s -X POST -H "Content-Type: application/json" -d "${input_data}" "${graylog_api}/system/inputs"
+    curl -sf -X POST -H "Content-Type: application/json" -d "${input_data}" "${graylog_api}/system/inputs" || errors="${errors}failed to create ${input_name} input, "
```

Comment:
```
errors=${errors}:"failed to create ${input_name} input, "
```
no reason for that comment, it is just how I would expect to see an append.

Reply:
The script runs in ash (default alpine shell). Still, I haven't seen that syntax before, do you have a documentation source? Does the colon has some special meaning?

Reply:
you are right, confusing it for bash